python/pyfory/tests/test_cross_language.py

```diff
@@ -338,6 +338,51 @@ def test_cross_language_serializer(data_file_path):
         f.write(new_buf.get_bytes(0, new_buf.writer_index))
 
 
+@cross_language_test
+def test_cross_language_serializer_go(data_file_path):
```

Comment:
why need a new test for go specially?

Reply:
We should use same tests as java used. Otherwise, we still don't get full xlang serialization features